lib/saluki-event/src/service_check/status.rs

```diff
@@ -0,0 +1,15 @@
+/// ServiceCheckStatus is the status type for service checks
```

Comment:
I know that in Go it's common to start a doc comment with the type/method name -- perhaps even enforced by the linter? -- but in Rust, we _generally_ prefer human-readable descriptions, which also means we try to avoid "stuttering", where we repeat the name of the thing we're describing.

So in this case, we know from the type name that we're dealing with the status of the service check, so we really don't need to say much more than "Service status."